src/AssetRegistry/AbstractPricingModule.sol

```diff
@@ -40,6 +40,11 @@ abstract contract PricingModule is Ownable {
         uint256 baseCurrency;
     }
 
+    struct WhiteListed {
```

Comment:
Do we need a new struct for this? also name sucks, no verbs for variable named

Reply:
renamed, struct better stays (packed and needs to be read together)